Fix sentence structure

Anirban166 opened the pull request:
Fixed grammar and sentence structure on appropriate instances.

Anirban166 committed:
Fix sentence structure

Fixed grammar and sentence structure on appropriate instances.

rust-highfive commented:
Thanks for the pull request, and welcome! The Rust team is excited to review your changes, and you should hear from @nikomatsakis (or someone else) soon.

If any changes to this PR are deemed necessary, please add them as extra commits. This ensures that the reviewer can see what has changed since they last reviewed the code. Due to the way GitHub handles out-of-date commits, this should also make it reasonably obvious what issues have or haven't been addressed. Large or tricky changes may require several passes of review and changes.

Please see [the contribution instructions](https://github.com/rust-lang/rust/blob/master/CONTRIBUTING.md) for more information.

nikomatsakis commented on README.md:
```diff
@@ -1,7 +1,7 @@
 
 This is the main source code repository for [Rust]. It contains the compiler,
-standard library, and documentation.
+standard library and documentation.
```
heh, I prefer the Good Ol' Oxford comma, but I'm happy either way.

steveklabnik replied:
we generally do use the oxford comma

Anirban166 replied:
okay, won't change the oxford ways then :)

nikomatsakis commented on README.md:
```diff
@@ -17,8 +17,8 @@ Read ["Installation"] from [The Book].
 
-The Rust build system has a Python script called `x.py` to bootstrap building
-the compiler. More information about it may be found by running `./x.py --help`
+The Rust build system has a Python script called `x.py` to bootstrap when building
```
I'm not sure about this word "when", it sounds sort of strange to me...

Anirban166 replied:
True, but "bootstrap building the compiler" doesn't sound quite right either, so what would you suggest in place of that?

nikomatsakis replied:
It doesn't bother me, but I never thought that carefully about it. I'm not sure we even need the word bootstrap in there, we could just say "The rust build system uses a Python script called `x.py` to build the compiler." If we want to, we could add "x.py manages the bootstrapping process."

Anirban166 replied:
That makes sense, will change it to that!

nikomatsakis commented on README.md:
```diff
@@ -54,9 +54,9 @@ or reading the [rustc dev guide][rustcguidebuild].
-    in the `[install]` section to a directory that you have write permissions.
+    It is recommended that you set the `prefix` value in the `[install]` 
+    section to a directory that you have write permissions if you plan to use 
+    the Rust build system to create an installation. (using `./x.py install`)
```
I agree the old sentence is off, but this one doesn't quite read right to me either. Maybe:

If you plan to use `x.py install` to create an installation, it is recommended that you set the `prefix` value in the `[install]` section to a directory.

Anirban166 replied:
That suits better, I'll change it!

steveklabnik commented on README.md:
```diff
@@ -1,7 +1,7 @@
-# The Rust Programming Language
+# The Rust Programming Language 
```
this change seems extraneous.

Anirban166 replied:
Sorry this was a misclick on my part.
But now that I think of it, a Rust logo wouldn't be bad. Would you like me to create a logo for it?
or like use some badge such as [this](https://img.shields.io/badge/--black?style=for-the-badge&logo=rust) for instance?

nikomatsakis replied:
badge seems nice

Anirban166 replied:
:D Will add that!

Anirban166 replied:
I've replaced the h1 heading "The Rust Programming Language" with the badge which says "Rust" beside the Rust logo, with a black background wording of "Programming Language", which you can adjust it to any wording you want by replacing the word before the first dash and after it like `badge/word1-word2-colour?` in the link `https://img.shields.io/badge/Rust-Programming%20Language-black?style=flat&logo=rust` and thereafter select the color you want. (hexcodes work too I think)

I have included an image link instead of the badge notation so you can tweak with parameters such as height width etc upto your preference.

nikomatsakis replied:
![image](https://user-images.githubusercontent.com/155238/85696928-5af4ae80-b6a7-11ea-81c6-a56e2cc0665d.png)

Anirban166 committed:
Minor correction to sentence structure

Anirban166 committed:
Update README.md

nikomatsakis commented on README.md:
```diff
@@ -1,7 +1,7 @@
-# The Rust Programming Language
+<img height = "50" src = "https://img.shields.io/badge/Rust-Programming%20Language-black?style=flat&logo=rust">
```
we should have an alt tag, I imagine

Anirban166 replied:
Sure, I'll add that now

Anirban166 replied:
I just previewed the changes in my mobile and it looked kinda cramped up so I set up responsiveness based on device width (height will be adjusted automatically)
I currently have it set to 90% in width of the viewing device, which you can change as per your convenience

Anirban166 replied:
@steveklabnik Do the changes seem okay or should I revert/change them?

nikomatsakis commented on README.md:
```diff
@@ -17,9 +17,9 @@ Read ["Installation"] from [The Book].
-The Rust build system has a Python script called `x.py` to bootstrap building
-the compiler. More information about it may be found by running `./x.py --help`
-or reading the [rustc dev guide][rustcguidebuild].
+The Rust build system uses a Python script called x.py to build the compiler,
```
```suggestion
The Rust build system uses a Python script called `x.py` to build the compiler,
```

nikomatsakis reviewed (approved):
LGTM, @steveklabnik wdyt?

Anirban166 committed:
Commit suggestion

Co-authored-by: Niko Matsakis <niko@alum.mit.edu>

Anirban166 committed:
Add alternate text for rust logo image

Anirban166 committed:
Added clickable-link 

Linked the logo/svg to https://www.rust-lang.org/
(change if required)

Anirban166 committed:
Add responsiveness to logo

Added responsiveness to image logo based on device width (set at 90% of device width, which can be changed as per requirement)

steveklabnik reviewed (approved):
Thanks!

steveklabnik commented:
@bors: r+ rollup

bors commented:
:pushpin: Commit 4c33b7c8d69648edcc7f4f3d2edbf96c00060d1e has been approved by `steveklabnik`

<!-- @bors r=steveklabnik 4c33b7c8d69648edcc7f4f3d2edbf96c00060d1e -->
<!-- homu: {"type":"Approved","sha":"4c33b7c8d69648edcc7f4f3d2edbf96c00060d1e","approver":"steveklabnik"} -->
